docs/pages/environment_setup.md

```diff
@@ -11,9 +11,9 @@ When you are finished with this guide head on over to the [Plugins](./plugins_do
 ## Set Up a Python Virtual Environment
 When you first try to run the MTT program you may get the error:  
   
-File "pyclient/pymtt.py", line XX, in <module>  
-from SOMETHING import SOMETHING  
-ImportError: No module named SOMETHING    
+File "pyclient/pymtt.py", line XX, in <module>    
+from SOMETHING import SOMETHING   
+ImportError: No module named SOMETHING      
```

Comment:
The only change here is extra whitespace at the end of each line?

Reply:
We talked about this over the phone -- extra 2 spaces at the end makes it start a new line